src/docs/cookbook/animation/opacity-animation.md

```diff
@@ -210,3 +206,7 @@ class _MyHomePageState extends State<MyHomePage> {
 ```
 
 ![Fade In and Out Demo](/images/cookbook/fade-in-out.gif){:.site-mobile-screenshot}
+
+[AnimatedOpacity]: {{site.api}}/flutter/widgets/AnimatedOpacity-class.html
```

Comment:
This should either be [`AnimatedOpacity`] or the references above should be fixed.